### DISCUSSION TITLE
Rivet high CPU usage

### DISCUSSION THREAD

**Discussion by adp60** [Ironclad/rivet#197] [General]
Just updated to version 1.4 on windows and I have noticed that just opening a graph makes my CPU fan spin up. This is without even running a graph. Task manager shows CPU usage at 20% on a Core i9 9900. What could be the issue?

**Comment by abrenneke** [comment]
Not sure, I'll double check that it's not doing anything dumb! Always trying to improve performance of Rivet, it's tough.

**Reply by adp60** [reply]
Thanks. Reason I was asking is because I am using Llamma cpp and need every cycle from my CPU.  There is a subprocess? called webview which seems to be the main culprit.

**Reply by abrenneke** [reply]
Yeah it does seem to be using a lot even at idle. Thanks for the report..
